cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/update/IidmToCgmes.java

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cgmes.conversion.update;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.powsybl.cgmes.model.CgmesSubset;
+import com.powsybl.iidm.network.Identifiable;
+
+/**
+ * @author Elena Kaltakova <kaltakovae at aia.es>
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ */
+public class IidmToCgmes {
+
+    public List<TripleStoreChange> convert(IidmChange change) {
+        if (change instanceof IidmChangeUpdate) {
+            return convertUpdate((IidmChangeUpdate) change);
+        } else {
+            return Collections.emptyList();
```

Comment:
Is it a programming issue if change is not a IidmChangeUpdate object? Should we throw an exception or at least warn?

Reply:
You are right, for the moment these are changes that are not supported. Should thrown an UnsupportedOperationException.

Reply:
Done